### PR TITLE
Remove product reference

### LINKED ISSUE #75: References to non-affiliated products / technology in scoping.md
in A.1.1. Software TEE there is a mention of "Other examples exist from other organizations not affiliated with the CCC."

QEMU, mentioned in that section, is not affiliated with the CCC.

The reader may make incorrect guesses about what products / technology are being referred to in the section.

Perhaps it would be better to either remove the "other examples" reference, or be more explicit about what examples you mean?

### PR DESCRIPTION
**Pull request by dthaler**
Fixes #75

Signed-off-by: Dave Thaler <dthaler@microsoft.com>

### CODE REVIEW

File: scoping.md [hunk 1]
```diff
@@ -286,22 +286,14 @@ There are multiple products and solutions today that are software-based and
...
-
-* **[Virtual Secure Mode (VSM)](https://azure.microsoft.com/en-us/blog/introducing-azure-confidential-computing/)**: a software-based TEE that's
-  implemented by Hyper-V in Windows 10 and Windows Server 2016. Hyper-V 
-  prevents administrator code running on the computer or server, as well
-  as local administrators and cloud service administrators from viewing
-  the contents of the VSM enclave or modifying its execution.
+As one example:
 
 * **[QEMU](https://www.linaro.org/blog/arm-trustzone-qemu/) ("quick emulator")**: very widely used open source machine emulator. ... Developers can use 
   the QEMU Arm Security Extensions to develop and work with Trusted
```
Comment (dthaler) [inline scoping.md:292]:
```suggestion
  the QEMU Security Extensions to develop and work with Trusted
```

File: scoping.md [hunk 2]
```diff
@@ -286,22 +286,14 @@ There are multiple products and solutions today that are software-based and
...
-Some examples include:
-
-* **[Virtual Secure Mode (VSM)](https://azure.microsoft.com/en-us/blog/introducing-azure-confidential-computing/)**: a software-based TEE that's
-  implemented by Hyper-V in Windows 10 and Windows Server 2016. Hyper-V 
-  prevents administrator code running on the computer or server, as well
-  as local administrators and cloud service administrators from viewing
-  the contents of the VSM enclave or modifying its execution.
+As one example:
 
 * **[QEMU](https://www.linaro.org/blog/arm-trustzone-qemu/) ("quick emulator")**: very widely used open source machine emulator. ... Developers can use
```
Comment (dthaler) [inline scoping.md:291]: Replace with github link?
```suggestion
* **[QEMU](https://www.linaro.org/blog/arm-trustzone-qemu/")**: very widely used open source machine emulator. ... Developers can use 
```

File: scoping.md [hunk 3]
```diff
@@ -286,22 +286,14 @@ There are multiple products and solutions today that are software-based and
...
-* **[Virtual Secure Mode (VSM)](https://azure.microsoft.com/en-us/blog/introducing-azure-confidential-computing/)**: a software-based TEE that's
-  implemented by Hyper-V in Windows 10 and Windows Server 2016. Hyper-V 
-  prevents administrator code running on the computer or server, as well
-  as local administrators and cloud service administrators from viewing
-  the contents of the VSM enclave or modifying its execution.
-
-* **[QEMU](https://www.linaro.org/blog/arm-trustzone-qemu/) ("quick emulator")**: very widely used open source machine emulator. ... Developers can use 
-  the QEMU Arm Security Extensions to develop and work with Trusted
+* **[QEMU](https://www.linaro.org/blog/arm-trustzone-qemu/")**: very widely used open source machine emulator. ... Developers can use 
+  the QEMU Security Extensions to develop and work with Trusted
```
Comment (dthaler) [inline scoping.md:292]:
```suggestion
  the QEMU security extensions that emulate a given TEE to develop and work with Trusted
```

File: scoping.md [hunk 4]
```diff
@@ -286,22 +286,14 @@ There are multiple products and solutions today that are software-based and
...
 
-* **[Virtual Secure Mode (VSM)](https://azure.microsoft.com/en-us/blog/introducing-azure-confidential-computing/)**: a software-based TEE that's
-  implemented by Hyper-V in Windows 10 and Windows Server 2016. Hyper-V 
-  prevents administrator code running on the computer or server, as well
-  as local administrators and cloud service administrators from viewing
-  the contents of the VSM enclave or modifying its execution.
-
-* **[QEMU](https://www.linaro.org/blog/arm-trustzone-qemu/) ("quick emulator")**: very widely used open source machine emulator. ... Developers can use 
-  the QEMU Arm Security Extensions to develop and work with Trusted
+* **[QEMU](https://www.linaro.org/blog/arm-trustzone-qemu/")**: very widely used open source machine emulator. ... Developers can use
```
Comment (dthaler) [inline scoping.md:291]:
```suggestion
* **[QEMU](https://github.com/qemu/QEMU)**: very widely used open source machine emulator. ... Developers can use 
```

File: scoping.md [hunk 5]
```diff
@@ -286,22 +286,14 @@ There are multiple products and solutions today that are software-based and
...
-  prevents administrator code running on the computer or server, as well
-  as local administrators and cloud service administrators from viewing
-  the contents of the VSM enclave or modifying its execution.
-
-* **[QEMU](https://www.linaro.org/blog/arm-trustzone-qemu/) ("quick emulator")**: very widely used open source machine emulator. ... Developers can use 
-  the QEMU Arm Security Extensions to develop and work with Trusted
+* **[QEMU](https://www.linaro.org/blog/arm-trustzone-qemu/")**: very widely used open source machine emulator. ... Developers can use 
+  the QEMU Security Extensions to develop and work with Trusted
   Execution Environments (TEEs) that are likely to be the primary consumers
   of the added functionality. Secure applications can then be developed
```
Comment (dcmiddle) [inline scoping.md:294]: Strike "secure applications can then be developed on the added TEEs without the need for dedicated hardware."
The goal of the section is to say there are s/w emulators outside of our definition of a TEE. Whether or not an application is secure in using an emulator seems like unnecessary text.

**Review by dcmiddle — CHANGES_REQUESTED** [review]

File: scoping.md [hunk 6]
```diff
@@ -286,22 +286,14 @@ There are multiple products and solutions today that are software-based and
...
-  as local administrators and cloud service administrators from viewing
-  the contents of the VSM enclave or modifying its execution.
-
-* **[QEMU](https://www.linaro.org/blog/arm-trustzone-qemu/) ("quick emulator")**: very widely used open source machine emulator. ... Developers can use 
-  the QEMU Arm Security Extensions to develop and work with Trusted
+* **[QEMU](https://github.com/qemu/QEMU)**: very widely used open source machine emulator. ... Developers can use 
+  the QEMU security extensions that emulate a given TEE to develop and work with Trusted
   Execution Environments (TEEs) that are likely to be the primary consumers
   of the added functionality. Secure applications can then be developed
   on the added TEEs without the need for dedicated hardware.
```
Comment (dthaler) [inline scoping.md:295]:
```suggestion
  of the added functionality.
```

**Comment by dthaler** [comment]
Merging per approval in TAC meeting today.

**Pull request merged**